dialogs/graph_settings.py

```diff
@@ -84,16 +99,20 @@ def accept_settings(self):
         self.parent.transpose_axes = \
             self.transposeAxesCheckBox.checkState() == QtCore.Qt.Checked
         self.parent.color_scheme = self.color_scheme
-        if self.radio_range_auto.isChecked():
-            self.parent.axes_range_mode = 0
-        elif self.radio_range_manual.isChecked():
+        
+        if self.radio_range_auto.isChecked(): 
+            self.parent.axes_range_mode = 0    
+                                                        
+         
+        elif self.radio_range_manual.isChecked():       
             self.parent.axes_range_mode = 1
-        x_range_min = self.spin_range_x_min.value()
-        x_range_max = self.spin_range_x_max.value()
-        y_range_min = self.spin_range_y_min.value()
-        y_range_max = self.spin_range_y_max.value()
+            x_range_min = self.spin_range_x_min.value()
+            x_range_max = self.spin_range_x_max.value()
+            y_range_min = self.spin_range_y_min.value()
+            y_range_max = self.spin_range_y_max.value()
+            self.parent.axes_range = [(x_range_min, x_range_max),
+                                      (y_range_min, y_range_max)]  
 
-        self.parent.axes_range = [(x_range_min, x_range_max),
-                                  (y_range_min, y_range_max)]
+        
         self.parent.on_draw()
```

Comment:
```suggestion
        self.parent.on_draw()
```
Code style issue:
- too many blank lines (2)